frontend/src/modules/history/components/Translation.css

```diff
@@ -3,13 +3,33 @@
     cursor: pointer;
     padding: 5px 10px 20px 10px;
     transition: all 0.1s ease-out;
+    display: flex;
 }
 
 .history .translation:hover {
     background: #4D5967;
 }
 
-.history .translation > header {
+.history .translation .avatar {
+    padding-right: 8px;
+    position: relative;
+    top: 3px;
+}
+
+.history .translation .avatar img {
+    border-radius: 6px;
+    border: 1px solid #5E6475;
```

Comment:
You will hate me for this I know. :) Could you set the border to 2px instead of 1px? It's against the spec, but also looks inconsistent with the rest of the UI, where we always use 2px.

Reply:
Fixed :)
Actually the opposite, I appreciate your code review. That's a great way to learn. Thank you for that!